test/fulfillmentSpec.js

```diff
@@ -348,6 +349,12 @@ describe('PUT /fulfillment', function () {
     })
 
   it('should not double spend when transfer is executed multiple times', function * () {
+    // Pre-fund hold account to allow double spending of source transfer
+    // This could happen if, for ex, 2 txn are escrowed, then first is executed twice.
```

Comment:
Not sure what you mean by "transaction" (transfer? database transaction?). We had a separate issue we fixed with the notary, where it was sending two fulfillments per transfer. If we only get one fulfillment per transfer, this won't happen, but I think it's good to have a defensive test to confirm the ledger behaves properly, even in that case.